### PR TITLE
fix: limit grip events to 1 hour difference

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
### Description of PR

Added a condition to limit grip events association to 1 hour time window.

What component(s) does this PR affect?

  - [ ] Back-End (Node.js, Next.js)
  - [x] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
  Does the PR require changes on other components? If yes, please mark the components:
  
  - [ ] Back-End (Node.js, Next.js)
  - [ ] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
### Related Issue
  
  
  
  
  
  
  Resolves #
  
### Type
  
  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to change)
  - [ ] Docs update
  - [ ] None of the above
  
  
  
### Checklist:
  
  - [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
  - [ ] This change requires a change in the documentation.
  - [ ] I have updated the documentation accordingly.
  >)`>>>)>

### CODE REVIEW

**Review by vkotronis — APPROVED** [review]
LGTM, please remove `console.log` statements if we do not need them

File: apps/artemis-web/components/hijack-info/hijack-info.tsx [hunk 1]
```diff
@@ -75,9 +76,27 @@ class HijackInfoComponent extends Component<any, any> {
...
+      const eventData = [];
 
       if (json.recordsTotal > 0) {
-        this.setState({ gripState: true, event_data: json.data, gripFetched: true });
+        json.data.forEach(event => {
+          if (Math.abs(new Date(hijackDataState["time_started"]).getTime() - Math.abs(parseInt(event.view_ts, 10) * 1000)) < 3600000) {
+            eventData.push(event);
+          }
+        });
+        console.log(eventData)
```
Comment (vkotronis) [inline apps/artemis-web/components/hijack-info/hijack-info.tsx:87]: TODO: remove

**Review by vkotronis — CHANGES_REQUESTED** [review]
Please merge the current master branch so that we see only the actual changes in the PR

**Comment by CuriouzK0d3r** [comment]
> Please merge the current master branch so that we see only the actual changes in the PR

I synced :)

**Review by vkotronis — APPROVED** [review]
LGTM

**Pull request merged**